go.mod

```diff
@@ -3,11 +3,18 @@ module github.com/nats-io/nats.go
 go 1.19
 
 require (
+	github.com/golang/protobuf v1.4.2
```

Comment:
@masumomo Thanks for the contribution! 🙌  Could you revert the changes to `go.mod` and `go.sum`? You can instead use the following `go_test.mod` like this for running the tests to avoid including the server into the main `go.mod`:

```
go test ./... --count=1 -p=1 -modfile=go_test.mod
```

Reply:
Hello Wally! Oh, Thank you so much! I realised we have `go_test.mod` in the repo and an instruction on README to run the test 🙏  I'll revert it!